morphic.js

```diff
@@ -4998,7 +5006,15 @@ CursorMorph.prototype.initializeClipboardHandler = function () {
         },
         false
     );
-    
+
+    this.clipboardHandler.addEventListener(
+        'keyup',
+        function (event) {
```

Comment:
Hmm....  I think I can see why we need this, but I'm not happy with crawling up the Morphic tree all the way to the root on every single keystroke. Maybe we can capture the ```World``` in a closure somewhere. Need to think about this. I hate shadowing the DOM. See, Web-Programming is all evil :-)